tools/extras/install_morfessor.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright  2017  Atlas Guide (Author : Lucas Jo)
+#
+# Apache 2.0
+#
+
+echo "#### installing morfessor"
+dirname=morfessor
+if [ ! -d ./$dirname ]; then
+	mkdir -p ./$dirname
```

Comment:
could you please change the tabs to spaces (each tab = two spaces)?

Reply:
applied retab with size 2